Description: do not strip binary in make step
 Binary stripping can be done later (so the debug symbols go into the
 dbgsym packages).

--- a/bspc/Makefile
+++ b/bspc/Makefile
@@ -91,7 +91,6 @@ release: $(EXEC)
 
 $(EXEC): $(GAME_OBJS)
 	$(CC) -o $@ $(GAME_OBJS) $(LDFLAGS)
-	strip $@
 
 $(EXEC)_g: $(GAME_OBJS)
 	$(CC) -o $@ $(GAME_OBJS) $(LDFLAGS)
